core/sail/shacl/src/main/java/org/eclipse/rdf4j/sail/shacl/ShaclSailValidationException.java

```diff
@@ -61,6 +61,9 @@ public ValidationReport getValidationReport() {
 			while (!propertyShapes.isEmpty()) {
 				ValidationResult validationResult = new ValidationResult(propertyShapes.pop(),
 						invalidTuple.line.get(0));
+				if (invalidTuple.line.size() > 1) {
+					validationResult.SetAct(invalidTuple.line.get(1));
+				}
```

Comment:
Are we sure that index 1 will always be the actual value? I'm also not sure if this handles the use of sh:or and sh:and.